app/src/main/java/com/example/fittracker/mvp/models/MainActivityModel.java

```diff
@@ -1,7 +1,6 @@
 package com.example.fittracker.mvp.models;
```

Comment:
Package names must be in singular.. instead "models" it should be "model"

Reply:
Done